pkg/controller/linstorsatelliteset/linstorsatelliteset_controller.go

```diff
@@ -851,6 +887,33 @@ func newSatelliteDaemonSet(satelliteSet *piraeusv1.LinstorSatelliteSet, satellit
 							},
 							Resources: satelliteSet.Spec.Resources,
 						},
+						{
+							Name:            "drbd-prometheus-exporter",
```

Comment:
Do we want to make it optional whether to add this container etc?

Reply:
I was thinking about this, too. The problem with that is: how to determine when to turn it on. Since we use Helm, and helm does not replace CRDs, upgrading from an older version would mean you'd not get the benefit of monitoring, unless you take the extra steps to replace the CRDs.

That reminds me, I still need to update the helm values...

Reply:
I don't think we should decide based on the behavior of updated systems. There will presumably/hopefully be many more new systems set up in the future than there are existing systems. I would make it optional but enable it in our Helm values.

Enabling it for existing systems would just be a matter of replacing the CRDs and editing the CRs (with Helm), right?